examples/images_contours_and_fields/demo_bboximage.py

```diff
@@ -19,24 +19,11 @@
 # Create a BboxImage with Text
 # ----------------------------
 txt = ax1.text(0.5, 0.5, "test", size=30, ha="center", color="w")
-kwargs = dict()
-
-bbox_image = BboxImage(txt.get_window_extent,
-                       norm=None,
-                       origin=None,
-                       clip_on=False,
-                       **kwargs
-                       )
-a = np.arange(256).reshape(1, 256)/256.
-bbox_image.set_data(a)
-ax1.add_artist(bbox_image)
+ax1.add_artist(BboxImage(txt.get_window_extent, data=[range(256)]))
```

Comment:
Is there a point in passing an iterator in a list here? It is sort of nice that it can be done, but I guess that this will not always work? Maybe more pedagogical to do `list(range(256))`? Same below.

(I have no strong opinion as such. Pros and cons.)

Reply:
It has to be a 2d construct (basically image data).

Reply:
I went back to an explicit 2d numpy array.

Reply:
Don't see a 2D array here still.

Reply:
forgot to push, sorry, fixed.